app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherViewModel.kt

```diff
@@ -170,9 +171,29 @@ class TabSwitcherViewModel @Inject constructor(
             val data = tabRepository.tabSwitcherData.first()
             tabRepository.setAnnouncementDisplayCount(data.announcementDisplayCount + 1)
         }
+
+        pixel.fire(AppPixelName.TAB_MANAGER_REARRANGE_BANNER_DISPLAYED)
+    }
+
+    fun onFeatureAnnouncementCloseButtonTapped() {
+        dismissFeatureAnnouncementBanner()
+
+        pixel.fire(AppPixelName.TAB_MANAGER_REARRANGE_BANNER_MANUAL_CLOSED)
+    }
+
+    fun onTabDraggingStarted() {
+        dismissFeatureAnnouncementBanner()
+
+        pixel.fire(AppPixelName.TAB_MANAGER_REARRANGE_BANNER_AUTODISMISSED)
```

Comment:
Do we want to send this pixel even in cases where the banner was not shown?

Reply:
Fixed in [698a592](https://github.com/duckduckgo/Android/pull/4757/commits/698a592aab9bad8dacca473a20e3420903579112).